FiducialRegistrationWizard/MRML/vtkMRMLFiducialRegistrationWizardNode.cxx

```diff
@@ -49,8 +55,9 @@ vtkMRMLFiducialRegistrationWizardNode::vtkMRMLFiducialRegistrationWizardNode()
   this->AddNodeReferenceRole( FROM_FIDUCIAL_LIST_REFERENCE_ROLE, NULL, fiducialListEvents.GetPointer() );
   this->AddNodeReferenceRole( TO_FIDUCIAL_LIST_REFERENCE_ROLE, NULL, fiducialListEvents.GetPointer() );
   this->AddNodeReferenceRole( OUTPUT_TRANSFORM_REFERENCE_ROLE );
-  this->RegistrationMode = "Rigid";
-  this->UpdateMode = "Automatic";
+  this->RegistrationMode = REGISTRATION_MODE_RIGID;
+  this->UpdateMode = UPDATE_MODE_MANUAL;
```

Comment:
this->UpdateMode = "Automatic" => this->UpdateMode = UPDATE_MODE_MANUAL

Why is the change?

Reply:
Possibly a typo on my part. Will look into it.